lms/djangoapps/courseware/tests/test_i18n.py

```diff
@@ -190,12 +219,16 @@ def test_preview_precedence(self):
         # Set user language preference
         set_user_preference(self.user, LANGUAGE_KEY, 'ar')
         # Verify preview-lang takes precedence
-        response = self.client.get('{}?preview-lang=eo'.format(self.url))
+        self.client.post(self.preview_language_url, {'preview_lang': 'eo', 'set_language': 'set_language'})
+        response = self.client.get(self.url)
+
         self.assert_tag_has_attr(response.content, "html", "lang", 'eo')
         # Hitting another page should keep the dark language set.
         response = self.client.get(reverse('courses'))
```

Comment:
I updated this, it now hits a different page  

The previous checks now use the same url as the other tests '/dashboard'.  I left the 'courses' here